tests/integ/test_pytorch_train.py

```diff
@@ -160,8 +157,9 @@ def test_deploy_model_with_accelerator(sagemaker_session, cpu_instance_type):
     pytorch = PyTorchModel(
         model_data,
         "SageMakerRole",
-        framework_version="1.3.1",
         entry_point=EIA_SCRIPT,
+        framework_version="1.3.1",
```

Comment:
Same here, why is it "1.3.1" instead of using `pytorch.defaults.LATEST_VERSION`?

Reply:
it's EI